spec/relaton_ccsds/data_parser_spec.rb

```diff
@@ -11,24 +11,10 @@
     let(:identifier) { "CCSDS 121.0-B-3" }
     subject { RelatonCcsds::DataParser.new doc, docs }
 
-    it "#parse" do
-      expect(subject).to receive(:parse_title).and_return :title
-      expect(subject).to receive(:parse_docid).and_return :docid
-      expect(subject).to receive(:parse_abstract).and_return :abstract
-      expect(subject).to receive(:parse_doctype).and_return :doctype
-      expect(subject).to receive(:parse_date).and_return :date
-      expect(subject).to receive(:parse_docstatus).and_return :docstatus
-      expect(subject).to receive(:parse_link).and_return :link
-      expect(subject).to receive(:parse_edition).and_return :edition
-      expect(subject).to receive(:parse_relation).and_return :relation
-      expect(subject).to receive(:parse_editorialgroup).and_return :editorialgroup
-      expect(subject).to receive(:parse_technology_area).and_return :technology_area
-      expect(RelatonCcsds::BibliographicItem).to receive(:new).with(
-        title: :title, docid: :docid, abstract: :abstract, doctype: :doctype, date: :date,
-        docstatus: :docstatus, link: :link, edition: :edition, relation: :relation,
-        editorialgroup: :editorialgroup, technology_area: :technology_area
-      ).and_return :bibitem
-      expect(subject.parse).to eq :bibitem
+    context "#parse" do
+      subject { RelatonCcsds::DataParser.new(doc, docs).parse }
+      it { is_expected.to be_a(RelatonCcsds::BibliographicItem) }
+      it { expect(subject.docidentifier.first.id).to eq(identifier) }
```

Comment:
Please add tests to be sure all the `parse_*` methods are called from the `parse` method.

Reply:
@andrew2net added more tests